src/styles/shared/_skeleton.scss

```diff
@@ -16,19 +13,6 @@ $thumbnail-sizes: (
   @return map-get($thumbnail-sizes, $size);
 }
 
-@mixin skeleton-shimmer {
```

Comment:
This would be considered a breaking change right? I know I've seen this used in web.

Reply:
Of course it is...